Maquetado/scss/titles.scss

```diff
@@ -1,16 +1,9 @@
-$urlImagen: "./assets/triangle.svg";
+$urlImagen: './assets/triangle.svg';
```

Comment:
use english for variable names, and also use spinal- case. `$image-url` would be a good choice here.